server/network.js

```diff
@@ -470,9 +470,11 @@ network.listNetwork_ = function(networkPath) {
  */
 network.incrementalEdges_ = function(file, genes, nodes) {
   var oldNodes = {};
-  nodes.forEach(function(node) {
-    oldNodes[node.id] = true;
-  });
+  if (nodes.length) {
```

Comment:
Really? What is the reason that it breaks? From the code it seems to work exactly the same way.